site/content/en/latest/tasks/extensibility/wasm.md

```diff
@@ -147,20 +147,9 @@ Verify the EnvoyExtensionPolicy status:
 kubectl get envoyextensionpolicy/http-wasm-source-test -o yaml
 ```
 
-### Testing
+## Testing
 
-Ensure the `GATEWAY_HOST` environment variable from the [Quickstart](../../quickstart) is set. If not, follow the
-Quickstart instructions to set the variable.
-
-```shell
-echo $GATEWAY_HOST
-```
-
-Send a request to the backend service:
-
-```shell
-curl -i -H "Host: www.example.com" "http://${GATEWAY_HOST}"
-```
+{{< boilerplate testing-the-configuration >}}
```

Comment:
Hi @daixiang0 Can you also modify other tasks in the same way in this PR?

Reply:
Could merge it first?

I need much time to check tasks one by one to modify.

Reply:
Prefer to use the same pattern for all the tasks. You can simply search and replace.

Reply:
@zhaohuabing done.